src/typ/tests.rs

```diff
@@ -44,35 +44,35 @@ fn field_map_reorder_test() {
             CallArg {
                 location: Default::default(),
                 label: None,
-                value: int(1),
+                value: int("1".to_string()),
```

Comment:
We could make this `int` function take `&str` instead of `String` and then call `.to_string()` in the body of the function so that we don't need to repeate `to_string()` where it is used

Reply:
I'm not sure if I'm getting it right but
`fn infer_int(value: &str, location: SrcSpan) -> Result<TypedExpr, Error> {
    Ok(TypedExpr::Int {
        location,
        value: value.to_string(),
        typ: int(),
    })
}`
does not do the thing (typ.rs)

Reply:
Here is the function: https://github.com/gleam-lang/gleam/blob/3f3000756e99a218d651edbf3e25126b66c2edf8/src/typ/tests.rs#L5-L8

I've written it in the closure syntax for some reason. Can just change `value,` to `value: value.to_string()` I think

Reply:
Oh, I see. Done